test-data/reposerver.py

```diff
@@ -0,0 +1,392 @@
+import argparse
```

Comment:
It would be nice if you can add where you got this from and, what the license is. It would also be nice if you move it to a separate directory with a readme maybe? Instead of having it in the "root" of the test-data dir.